Nodejs/Product/Nodejs/Snippets/1033/Nodejs/tryc.snippet

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>tryc</Title>
+      <Description>Code Snippet for a Try...Catch Statement</Description>
+      <Shortcut>tryc</Shortcut>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Code Language="Nodejs">
+        <![CDATA[try {
+
+} catch (error) {

```

Comment:
you probably want to make the error a replaceable parameter here since people are picky about what they name their errors. I believe it's also more common to use ```e``` rather than ```error```